### PR TITLE
WIP: Neuroscout related changes

### PR DESCRIPTION
**Pull request by adelavega**
- [x] `ModelSpecLoader` is not working working when the model was specified as a string, rather than extract from the filesystem using `BIDSLayout`. I fix that here
Not sure this will work w/ `auto_model`. Doesn't that return a dictionary (not a path, or json in string format)?
- [x]  I need to be able to specify more than a single derivatives dir (neuroscout event files, fmriprep files) Changing fitlins to allow multiple derivatives, as in `BIDSLayout`

### CODE REVIEW

**Comment by adelavega** [comment]
This is WIP I may pack a few other random things into this

**Comment by adelavega** [comment]
Regarding TR, it looks like the problem is that `get_metadata` does not go across derivatives (TR info is in `neuroscout`, image is in `fmiprep`.

**Comment by adelavega** [comment]
@effigies Let me know what you think about specifying arbitrary derivatives directories. They way I did it here is that you can keep passing derivatives to be indexed by pybids.

E.g. `fitlins [other args] --derivatives = /my/fmriprep --derivatives= /my/events --derivatives = /maybe/something else`
Those argument are put together into a list that is passed to `BIDSLayout`. If none is specified, the default is `True`, and all derivatives in `bids_dir/derivatives` are indexed. I suppose we can also allow `None` if the user doesn't want any derivatives to be indexed (although when would that be the case)?

Also, it would be syntactically nicer if we didn't have to repeated the `--derivatives` argument and just pass a list, but Im not sure how to do that w/ argparse.

File: fitlins/cli/run.py [hunk 1]
```diff
@@ -79,9 +79,9 @@ def get_parser():
                              'removed)')
     g_bids.add_argument('-m', '--model', action='store', default='model.json',
                         help='location of BIDS model description (default bids_dir/model.json)')
-    g_bids.add_argument('-p', '--preproc-dir', action='store', default='fmriprep',
-                        help='location of preprocessed data (if relative path, search '
-                             'bids_dir/derivatives, followed by output_dir)')
+    g_bids.add_argument('-d', '--derivatives', action='append',
```
Comment (effigies) [inline fitlins/cli/run.py]:
```suggestion
    g_bids.add_argument('-d', '--derivatives', action='store', nargs='+',
```

This requires all derivatives to be listed in the same place (`-d /path/a /path/b` will produce `['/path/a', '/path/b']`, while `-d /path/a -d /path/b` will produce `['/path/b']`), but we follow this idiom with `--participant-label`, as well.

You could do `append` and `nargs='+'`, but then you'd get a list of lists, which would need to be concatenated.

**Review by effigies — COMMENTED** [review]
LGTM. One comment on the argument parser.

**Comment by adelavega** [comment]
If you want go ahead and merge and I can open up a new PR for any forthcoming changes... or I can keep going here up to you.

**Pull request merged**